hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/DataChecksum.java

```diff
@@ -43,6 +49,9 @@
   public static final int CHECKSUM_CRC32C  = 2;
   public static final int CHECKSUM_DEFAULT = 3; 
   public static final int CHECKSUM_MIXED   = 4;
+
+  public static final Logger LOG = LoggerFactory.getLogger(DataChecksum.class);
```

Comment:
make private

Reply:
Made private. It was copied from the wrong place (1/3 cases in hadoop-common btw.)